tests/integration_test.rs

```diff
@@ -30,7 +30,7 @@ mod tests {
                 destinations::{Destination, DestinationType},
                 Operator, QueryCondition, TriggerCondition,
             },
-            dashboards::{v1, Dashboard, Dashboards},
+            dashboards::{v1, Dashboard},
```

Comment:
`Dashboards` no longer exists, nor is the new `ListDashboardsResponseBody` deserializable in the app since the app only needs to serialize it. Therefore I've made some updates to the e2e tests to account for this.